src/components/calcite-combobox-item/calcite-combobox-item.scss

```diff
@@ -83,6 +83,7 @@ ul {
 }
 
 .title {
+  @apply truncate;
```

Comment:
I don't think we should truncate the list items. I like that you can view the full text there, especially for mobile devices that can't rely on `title` attribute hover behavior. Can this line be removed? 

@bstifle do you agree or what should we do here? If it's removed, we get this:
![image](https://user-images.githubusercontent.com/42423180/122277326-9ed55a80-ce9a-11eb-955a-5a7da4031307.png)

Instead of this:
![image](https://user-images.githubusercontent.com/42423180/122277388-b3195780-ce9a-11eb-9081-950663e26e78.png)

Reply:
Fixed! makes sense.

Reply:
![image](https://user-images.githubusercontent.com/14083530/122289135-f9c97a80-ceb7-11eb-936a-b34b3f6d9ea8.png)
this is more ideal!